CIScripts/Test/Utils/IisTcpTestDockerImageBuild.ps1

```diff
@@ -0,0 +1,19 @@
+. $PSScriptRoot\..\..\Common\Aliases.ps1
+$DockerfilePath = "$PSScriptRoot\..\..\DockerFiles\iis-tcptest\Dockerfile"
+
+function Initialize-IisTcpTestDockerImage  {
+    Param ([Parameter(Mandatory = $true)] [PSSessionT] $Session)
+
+    $DockerFilesDir = "C:\DockerFiles"
+    Invoke-Command -Session $Session -ScriptBlock {
+        New-Item -ItemType Directory -Force $Using:DockerImagesDir | Out-Null
+    }
+
+    Write-Host "Copying iis-tcp-test image Dockerfile"
+    Copy-Item -ToSession $Session -Path $DockerfilePath -Destination $DockerFilesDir
```

Comment:
We think that this should copy not only dockerfile but whole directory 'iis-tcptest'